canonical-web-and-design/juju.is
# Build the about integration page

## QA
- Go to http://0.0.0.0:8041/about/integration
- Check the copy matches the [copy doc](https://docs.google.com/document/d/1xbMPHUB-vJ0G-TIozCarZXPDqhbQgf1EQb2lQlv6-3s/edit#)
- There are a number of illustrations requested in the copy doc.

Fixes https://github.com/canonical-web-and-design/juju.is/issues/96
